components/scream/src/share/io/tests/io.cpp

```diff
@@ -195,36 +213,56 @@ std::shared_ptr<FieldRepository<Real>> get_test_repo(const Int num_lcols, const
   FieldIdentifier fid1("field_1",FL{tag_h,dims_h},m,gn);
   FieldIdentifier fid2("field_2",FL{tag_v,dims_v},kg,gn);
   FieldIdentifier fid3("field_3",FL{tag_2d,dims_2d},kg/m,gn);
+  FieldIdentifier fid4("field_packed",FL{tag_2d,dims_2d},kg/m,gn);
 
   // Register fields with repo
+  using Spack        = ekat::Pack<Int,SCREAM_SMALL_PACK_SIZE>;
+  // Make sure packsize isn't bigger than the packsize for this machine, but not so big that we end up with only 1 pack.
+  const int packsize = 2;
+  using Pack         = ekat::Pack<Real,packsize>;
   repo->registration_begins();
   repo->register_field(fid1,{"output"});
   repo->register_field(fid2,{"output","restart"});
   repo->register_field(fid3,{"output","restart"});
+  repo->register_field<Pack>(fid4,{"output","restart"}); // Register field as packed
   repo->registration_ends();
 
+  // Make sure that field 4 is in fact a packed field
+  auto field4 = repo->get_field(fid4);
+  auto fid4_extent = field4.get_header().get_alloc_properties().get_last_extent();
+  auto fid4_final_dim = field4.get_header().get_identifier().get_layout().dims().back();
+  REQUIRE(fid4_extent-fid4_final_dim > 0);
```

Comment:
I think you need `>=0`. If pack sizes divides the field last dim, then `fid4_extent` and `fid4_final_dim` are the same.  In your case, you have nlevs=3 and a pack size 2, so it's fine. But if someone edits the test, it might be hard to catch what's going on.

You could make the test robust to edits, like this:
```
// Make sure that if ppl alter the test inputs, they see an error
EKAT_REQUIRE_MSG((last_dim % packsize)==0,
    "Error! Please, don't alter "nlevs" or "packsize" for this test.\n);
REQUIRE (extent-last_dim>0);
```

Reply:
you bring up a good point.  What happens when packsize = 1?  Shouldn't extent = last_dim?

Reply:
if so, then I think the REQUIRE shouldn't be there.  I'm essentially trying to avoid someone nullifying the packed test by accidentally making everything fit perfectly with nlev and packsize.